assets/translations/Russian

```diff
@@ -63,11 +63,11 @@ Delete=Удалить
 Description=Описание
 Directories=Каталоги
 Directory of the box art for the selector=Каталог с миниатюрами для показа в селекторе
-Displays last launched program's output=Отображает журнал последнего запущенного приложения
+Displays last launched program's output=Сообщения последнего запущенного ярлыка
```

Comment:
"Показывает сообщения" to different it from line 104?

Reply:
It might be good, but it won't fit in the allotted space. The trimming will happen here:
Показывает сообщения последнего запущенного я|рлыка
![screenshot](https://github.com/MiyooCFW/gmenu2x/assets/43220515/b7992a97-0a49-42e5-a86a-ff48f48bf461)

Unfortunately, I noticed this too late (I didn’t understand at first how to make Log Viewer entry appear in the menu).
After a long search of variants (and there were many of them), the most suitable replacements seemed to me to be those that appeared here.
The meaning (even if they are the same) is not lost and they fit into the designated places.
![screenshot0](https://github.com/MiyooCFW/gmenu2x/assets/43220515/fc43ad88-06c6-4391-977a-2e12e1cf500a)